tasks/prime.py

```diff
@@ -272,7 +274,80 @@ def on_start(self):
             name="/internal/service_members/{serviceMemberId}/backup_contacts",
             data=json.dumps(payload),
             headers={"content-type": "application/json"},
-            **self.user.cert_kwargs,
+        )
+
+        # Setup customer order
+        overrides = {
+            "service_member_id": service_member_id,
+            "issue_date": datetime.now().strftime("%Y-%m-%d"),
+            "report_by_date": datetime.now().strftime("%Y-%m-%d"),
+            "orders_type": "PERMANENT_CHANGE_OF_STATION",
+            "orders_type_detail": "HHG_PERMITTED",
+            "has_dependents": False,
+            "spouse_has_pro_gear": False,
+            "new_duty_station_id": stations[1]["id"],
+            "orders_number": None,
+            "tac": None,
+            "sac": None,
+            "department_indicator": None,
+        }
+        payload = self.fake_request("/orders", "post", INTERNAL_API_KEY, overrides, True)
+        order_resp = self.client.post(
+            self.customer_path("/internal/orders"),
+            data=json.dumps(overrides),
+            headers={"content-type": "application/json"},
+        )
+        order = order_resp.json()
+
+        document_id = order["uploaded_orders"]["id"]
+        upload_file = {"file": open(TEST_PDF, "rb")}
+        self.client.post(
+            self.customer_path(f"/internal/uploads?documentId={document_id}"),
+            name="/internal/uploads",
+            files=upload_file,
+        )
+
+        # Setup customer shipment
+        move_id = order["moves"][0]["id"]
+        self.client.patch(
+            self.customer_path(f"/internal/moves/{move_id}"),
+            name="/internal/moves/{moveId}",
+            data=json.dumps({"selected_move_type": "HHG"}),
+            headers={"content-type": "application/json"},
+        )
+
+        service_member = service_member_resp.json()
+        address = service_member["residential_address"]
+        address.pop("id")  # remove unneeded id
+        overrides = {
+            "moveTaskOrderID": move_id,
+            "shipmentType": "HHG",
+            "pickupAddress": address,
```

Comment:
This postal code will determine which TOO queue it gets routed to under the recent changes.